static/js/apps/eval_retrieval_generation/rag_ans_feedback.tsx

```diff
@@ -54,33 +58,37 @@ export function RagAnsFeedback(): JSX.Element {
     Promise.all([claimsCountPromise, falseClaimsCountPromise]).then(
       ([claimsCountData, falseClaimsCountData]) => {
         if (claimsCountData && falseClaimsCountData) {
-          setClaimsCount(Number(claimsCountData.toString()));
-          setFalseClaimsCount(Number(falseClaimsCountData.toString()));
-          setIsSubmitted(true);
+          setResponse({
+            claimsCount: Number(claimsCountData.toString()),
+            falseClaimsCount: Number(falseClaimsCountData.toString()),
+            isSubmitted: true,
+          });
         } else {
-          setClaimsCount(0);
-          setFalseClaimsCount(0);
-          setIsSubmitted(false);
+          setResponse({
+            claimsCount: 0,
+            falseClaimsCount: 0,
+            isSubmitted: false,
+          });
         }
       }
     );
   }, [sheetId, sessionQueryId, sessionCallId]);
 
   const checkAndSubmit = async (): Promise<boolean> => {
-    if (isSubmitted) {
+    if (response.isSubmitted) {
```

Comment:
check null first?

Reply:
we checked null before rendering the components that make this call, but added a check just in case in the future we get rid of that check